Neos.Eel/Classes/FlowQuery/Operations/Object/PropertyOperation.php

```diff
@@ -49,17 +49,17 @@ class PropertyOperation extends AbstractOperation
      */
     public function evaluate(FlowQuery $flowQuery, array $arguments)
     {
-        if (!isset($arguments[0]) || empty($arguments[0])) {
+        if (!isset($arguments[0]) || empty($arguments[0]) || !is_string($arguments[0])) {
```

Comment:
Not 100% sure about this. Can this argument be for example an object that can be `toString()`'d?